include/slang.h

```diff
@@ -4404,6 +4405,11 @@ namespace slang
         virtual SLANG_NO_THROW void SLANG_MCALL setAllowGLSLInput(
             bool                    value) = 0;
 
+            /** Enable or disable an experimental compiler passes
+             */
```

Comment:
Functions should be added towards the end to avoid breaking binary compatibility. Btw, I don't think this is needed given that the user can just encode the option as an entry when creating the session.